api/verify.go

```diff
@@ -126,7 +126,7 @@ func (a *API) Verify(w http.ResponseWriter, r *http.Request) error {
 		}
 
 		if cookie != "" && config.Cookie.Duration > 0 {
-			if terr = a.setCookieToken(config, token.Token, cookie == useSessionCookie, w); terr != nil {
+			if terr = a.setCookieTokens(config, token, cookie == useSessionCookie, w); terr != nil {
```

Comment:
@kangmingtay based on the conversation last night with @thorwebdev, we always want to set the cookie and Thor suggested removing this check (and same in token, and signup).

Is this check still needed?

Reply:
Sorry, line is 128 -- the check on cookie and duration.

Reply:
@kangmingtay we'll need to remove it as it's impossible to provide the `x-use-cookie` header to the `/verify` route as it's called via the oauth redirect and not the gotrue-js client. I don't think there's any harm in always sending the cookie header, so I think we should remove.